src/server/routes.py

```diff
@@ -170,3 +170,24 @@ def redirect_old_hero_images(folder, image):
 @app.route('/static/css/2019.css')
 def redirect_old_css():
     return redirect("/static/css/almanac.css?%s" % request.query_string.decode()), 301
```

Comment:
Is this a safe remediation for the warning below? Probably ok to not forward params on an old redirect, not like caching is critical.

```suggestion
    return redirect("/static/css/almanac.css"), 301
```

Reply:
To be honest probably don't even need this anymore! But sure let's go with that.